scripts/start

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Start Umbrel
+
+if [[ $UID != 0 ]]; then
+    echo "Umbrel must be started as root"
+    echo "Please re-run this script as"
+    echo "  sudo ./scripts/start"
+    exit 1
+fi
+
+check_dependencies () {
+  for cmd in "$@"; do
+    if ! command -v $cmd >/dev/null 2>&1; then
+      echo "This script requires \"${cmd}\" to be installed"
+      exit 1
+    fi
+  done
+}
+
+check_dependencies fswatch readlink dirname docker docker-compose rsync jq wget git
```

Comment:
Why are these all added here?

For example it's not clear to me where or why `jq` is required.

And if it's removed from whichever script requires it we might forget to remove it here.

If they need to be added here because the other scripts will error later rather than at start time maybe we could add some comments here explaining which files require which deps.

Reply:
Ah yes, comments would be nice. `jq` is used by the OTA update trigger by the way. 

The reason I added them here was to make sure something doesn't go wrong **after** the user is up and running Umbrel, because for instance, let's say `jq` isn't installed on the host and the OTA update fails, there's no way the user would be able to know what went wrong.